tests/test_cli.py

```diff
@@ -130,3 +130,47 @@ def test_validate_args():
             "all",
         ]
     )
+
```

Comment:
If you are wondering "why isn't there a test case here checking that passing multiple `--file-key` together with `--output pyproject` is rejected?"... it's because I put the exception-raising code for that down further, in `make_dependency_file()`, so that the exception would also be raised if `rapids-build-backend` (which does not use the CLI) tried to pass inputs like that.

I did add a unit test in `test_rapids_dependency_file_generator` checking that that exception is raised as expected.